client/service-worker.js

```diff
@@ -0,0 +1,41 @@
+// The Lounge - https://github.com/thelounge/lounge
+/* global clients */
```

Comment:
Does this have a special meaning? It looks like a machine-readable comment, like ESLint does.